remoteworkitem/trackeritem_test.go

```diff
@@ -22,40 +23,44 @@ func (test *TestTrackerItemRepository) TestUpload() {
 	resource.Require(t, resource.Database)
 
 	test.DB.Exec(`DELETE FROM "tracker_items"`)
```

Comment:
This seems a bit drastic. Why do you need to delete all tracker items? And why can't you use the db test suite to delete all that it has created after a test?

Reply:
This is an old piece of code. `remoteworkitem` package needs a lot of refactoring.